test/offsetarray.jl

```diff
@@ -303,6 +310,10 @@ map!(+, dest, am, am)
 @test dest[1,8] == 4
 @test dest[1,9] == -2
 
+am = map(identity, a)
```

Comment:
hm, seems this whole file is written at global scope. that prevents gc from ever cleaning up the intermediate values. maybe we should defer this work and add this to various `@testsets` once kshyatt's PR is merged.